deutsche-bank-adapter/src/test/java/de/adorsys/xs2a/gateway/service/impl/mapper/DeutscheBankConsentInformationMapperTest.java

```diff
@@ -0,0 +1,50 @@
+package de.adorsys.xs2a.gateway.service.impl.mapper;
+
+import de.adorsys.xs2a.gateway.service.Links;
+import de.adorsys.xs2a.gateway.service.consent.AccountAccess;
+import de.adorsys.xs2a.gateway.service.consent.ConsentInformation;
+import de.adorsys.xs2a.gateway.service.consent.ConsentStatus;
+import de.adorsys.xs2a.gateway.service.impl.model.DeutscheBankConsentInformation;
+import org.junit.Test;
+import org.mapstruct.factory.Mappers;
+
+import java.time.LocalDate;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class DeutscheBankConsentInformationMapperTest {
+    private static final AccountAccess ACCOUNT_ACCESS = new AccountAccess();
+    private static final LocalDate VALID_UNTIL = LocalDate.of(2121, 12, 12);
+    private static final int FREQUENCY_PER_DAY = 10;
+    private static final LocalDate LAST_ACTION_DATE = LocalDate.of(2021, 10, 10);
+    private static final Links LINKS = new Links();
```

Comment:
What do you think about setting some data to `Links` to check, whether it maps as expected?

Reply:
It compares them by reference since `equals` is not overloaded. I don't think fiddling with content is necessary.
All fields have the same types and the "mapping" is needed only to adapt to the typo in the json property name.